atomic_reactor/plugins/sendmail.py

```diff
@@ -150,6 +150,10 @@ def _should_send(self, success, manual_canceled):
         return should_send
 
     def _get_image_name_and_repos(self):
+        # check_and_set_platforms plugin ran and created initiated build_dir
+        platforms = self.workflow.data.plugins_results.get(PLUGIN_CHECK_AND_SET_PLATFORMS_KEY)
+        if not platforms:
+            return '', []
```

Comment:
Instead of checking what check_and_set_platforms returned, could we directly check if the build dir is initialized?

Reply:
ok